tests/integration-tests/tests/ad_integration/test_ad_integration/test_ad_integration/ad_stack.yaml

```diff
@@ -267,14 +267,22 @@ Outputs:
           - ""
           - - Ref: AWS::StackName
             - DomainName
-  Password:
+  AdminPassword:
     Value: {{ ad_admin_password }}
     Export:
       Name:
         Fn::Join:
           - ""
           - - Ref: AWS::StackName
-            - Password
+            - AdminPassword
+  UserPassword:
+    Value: {{ ad_user_password }}
+    Export:
+      Name:
+        Fn::Join:
+          - ""
+          - - Ref: AWS::StackName
+            - UserPassword
```

Comment:
Is this a ref or the UserPassword output?

Reply:
This is the export name of CloudFormation stack outputs. The export name is used for cross stack reference. It is optional. I only added it to be consistent with the what we did for AdminPassword